### ISSUE TITLE
icon not rendered

### ISSUE THREAD

**Issue by sinofseven** [terrastruct/d2#354]
When I output png and svg samples of the document, the icons were not rendered correctly.
https://d2lang.com/tour/icons

![9005529033331653  スクリーンショット 2022-12-05 15 23 06](https://user-images.githubusercontent.com/13509891/205563790-2d774336-e153-48fa-a18f-7a01f1b5e50d.png)

test.d2
```
my network: {
    icon: https://icons.terrastruct.com/infra/019-network.svg
}
```

```bash
$ d2 test.d2 test.svg
$ d2 test.d2 test.png
$ d2 --version
0.0.13
```

test.svg
![test](https://user-images.githubusercontent.com/13509891/205562214-bacd0ff0-b59c-41e9-b06a-07da26cb7970.svg)

test.png
![test](https://user-images.githubusercontent.com/13509891/205562328-b7aba12d-62bd-4c3b-963e-5b87925329d3.png)

**Comment by alixander** [comment]
does it work in --watch? the current version embeds the URLs, so they are only viewable in a browser environment, but won't for example, in an embedded one. we added a `--bundle` flag and also fixed png exports to inline the image bytes directly in there. it'll be in the next release (coming tomorrow)!

**Comment by alixander** [comment]
actually i'll keep this open until we verify with you that the new version works

**Comment by mschirrmeister** [comment]
Just build the `master` branch, version `0.0.13.-HEAD` and that seems to work fine for me. Straight to png looks fine and I can see that the image is embedded in the svg file.

**Comment by schoenenberg** [comment]
I think there is a recursion.. Using version `0.2.1` same placeholder as above. Also on the online playground. Please reopen this issue!

![deployment_diagram](https://user-images.githubusercontent.com/9002242/221527696-543214a7-eb9f-4e87-937a-5e4ef751d485.png)

**Comment by alixander** [comment]
@schoenenberg there was a regression where local images weren't working (https://github.com/terrastruct/d2/issues/889), which is fixed and will be in the next release (should be today!). but it shouldn't affect icons locally. can you copy paste your script?

**Comment by schoenenberg** [comment]
@alixander sure. See below:

```
ds_a.db_a: DB A
ds_a.db_a.shape: cylinder
ds_b.db_b: DB B
ds_b.db_b.shape: cylinder
ds_a.db_c: DB C
ds_a.db_c.shape: page

ds_a: Datacenter A
ds_b: Datacenter B

ds_a.p_a: Process A
ds_b.p_b: Process B

ds_a.f_a: Public Frontend
ds_a.f_a.shape: hexagon

u_a: Data Owner A
u_a.shape: image
u_a.icon: https://icons.terrastruct.com/essentials%2F359-users.svg
u_b: Data Owner B
u_b.shape: image
u_b.icon: https://icons.terrastruct.com/essentials%2F359-users.svg

ds_a: {
  p_a
  db_a
  db_c
  f_a
}

ds_b: {
  p_b
  db_b
}

ds_a.p_a -> ds_b.p_b
ds_a.p_a -> ds_a.db_a
ds_b.p_b -> ds_b.db_b
ds_b.p_b -> ds_a.db_c

ds_a.f_a -> ds_a.p_a
ds_a.f_a -> ds_b.p_b

u_a -> ds_a.db_a
u_b -> ds_b.db_b
```

**Comment by alixander** [comment]
yeah this looks fine on playground. it's just that exports in playground don't bundle the images for PNG, that's a separate issue: https://github.com/terrastruct/d2-playground/issues/17 . but it should be perfectly fine run on CLI. ty @schoenenberg 

https://play.d2lang.com/?script=pJLPTsQgEMbvPMVcPC6baDzYg4l1s1d9gwbobLdJlxIG_BPju5uBQt3Gm6dmvvmVb_iGnjole92pBg4tPIlay3SRBsznNPIf3NHc0Yls1_ovMp1hEvm81oV0akCR5AYOKiiDNqDP7vpKajMmHaOvfjZItHDSMVu0Ap4SGPU0Gjj62Qa0fe0U_zN-qGG2QsQyAby82zRBXKnxwmOywK-lgXMIjpr9niuSAb1XFHw0QZr5skcitGFUE93cHu_uH3aR0JOkt0HEcqnFpWVl66L_67Lk-SUAXKcEAG8xf4wAOHVKfIsl4Qzp3NWLnlKG3SOUdLdifhg1_IqmMzZqXvi6lCq7fMJvMZtx0tdGcevxEwAA__8%3D&
